export/hgnc_symbol_map.csv

```diff
@@ -3804,6 +3805,11 @@ THRA,11796
 THRB,11799
 TIAM1,11805
 TIAM2,11806
+TIMM10,11814
+TIMM10B,4022

```

Comment:
is this a place to introduce a TIM10 gene family as an intermediate? is it either one or the other is necessary to make a tim 22 complex?